### DISCUSSION TITLE
Is it possible to not clone homebrew-core?

### DISCUSSION THREAD

**Discussion by davidji99** [Homebrew/discussions#1820] [Everyday usage]
Similar to https://github.com/Homebrew/discussions/discussions/1738, I'm wondering if `brew` can be configured to not clone homebrew-core during installation or tapping a private brew tap.

Context: I use `brew` to install tools from a private tap in a CI system. While CI system can support caching the homebrew directory, this directory is quite massive and I'd like to avoid uploading and downloading the cache for each run. In this scenario, I don't plan on using any tools from homebrew-core.

Thanks for your time.

**Comment by SMillerDev** [comment]
Caching is definitely the best option. If you don't cache it you're really just saving the upload anyway.
